src/services/ethereum/gas.ts

```diff
@@ -100,8 +101,7 @@ async function estimateGasSavings(): Promise<IGasSavingsResponse> {
 async function fetchEstimatedGasSavings(): Promise<IGasSavingsResponse> {
   const { gwei: minGasPrice } = await estimateGas('7d');
 
-  const network = ethers.providers.getNetwork(1);
-  const provider = ethers.getDefaultProvider(network);
+  const provider = getProvider(1);
```

Comment:
This is my laziness but you shouldn't do it like this - you would use the `enum` with the chain id here